site-src/geps/gep-922.md

```diff
@@ -63,7 +63,8 @@ responsibilities to ensure we're providing a consistent experience.
 * MAY include the following changes to an existing API version with a new bundle
  **patch** version:
     * Clarifications to godocs
-    * Loosening validation to fix a bug (not to enable a new feature)
+    * Updating validation and/or CRD annotations to fix a bug (not to enable a
+      new feature)
```

Comment:
Updates to CRDs and/or code to fix a bug

Why be specific to validation or annotations? (I could be missing something)

Reply:
Good call, updated.